docs/config_entries_config_flow_handler.md

```diff
@@ -249,12 +250,36 @@ async def async_migrate_entry(hass, config_entry: ConfigEntry):
     return True
 ```
 
+## Reconfigure
+
+A config entry can allow reconfiguration by adding a `reconfigure` step. This provides a way for integrations to allow users to change config entry data without the need to implement an `OptionsFlow` for changing setup data which are not meant to be optional.
+
+This is not meant to handle authentication issues or reconfiguration of such. For that we have the [`reauth`](#reauthentication) step which should be implemented.
```

Comment:
There's no way to start a reauth step by the user.

Reply:
That's not the intention it's implied it can be manually started. I'll extend the sentence a bit to clarify.